frontend/src/scenes/insights/insightLogic.tsx

```diff
@@ -241,7 +245,7 @@ export const insightLogic = kea<insightLogicType>({
                 const duration = new Date().getTime() - values.queryStartTimes[queryId]
                 const tags = {
                     insight: values.activeView,
-                    scene: sceneLogic.values.scene,
+                    scene: IS_TEST_MODE ? 'circular load if sceneLogic no loaded first' : sceneLogic.values.scene,
```

Comment:
This doesn't look great… Is there no other way?

Reply:
Well, removed, but this does boot up `sceneLogic` and as a result loads everything that the insightsScene needs, which is a lot more extra stuff that suddenly appears. I guess adding a `.isMounted()` to the logic will solve the issue.

Until then, tests are just a bit slower:

![image](https://user-images.githubusercontent.com/53387/134138359-12ca7216-d0fa-41b0-9ab3-5062f525f98f.png)